docs/how-tos/using-the-event-bus.rst

```diff
@@ -0,0 +1,87 @@
+Using the Open edX Event Bus
```

Comment:
I'm going to add more details about this, like console logs, etc, in another PR. Thanks!